### PR TITLE
Bug 1242615 - Create marionette js test that checks the contents of the info view 

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6301597c4aea7dded5e9d93ea36c703be32986c7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4754c170365b25a207045c92ce6d9bdec1414601

File: apps/gallery/test/marionette/lib/fullscreen_view.js [hunk 1]
```diff
@@ -46,7 +46,7 @@
     openSaveButton: '#save',
     fullScreenToolBar: '#fullscreen-toolbar',
     fullScreenToolBarOptions: '#fullscreen-toolbar > a',
-
+    infoButton: '#fullscreen-info-button-tiny',
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/lib/fullscreen_view.js]: nit: we can get rid of comma at the end

File: apps/gallery/test/marionette/info_view_test.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    app = new Gallery(client);
+    fullscreen_view = new Fullscreen_View(client);
+    info_view = new GalleryInfoView(client);
+    app.launch();
+  });
+
+  test('check displayed information of image', function() {
+    assert.strictEqual(app.thumbnails.length, 1);
+    app.tapFirstThumbnail();
+    assert.ok(fullscreen_view.displayed);
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/info_view_test.js]: To be safer can we add wait condition here to check if infoButton is displayed

**Comment by punamdahiya** [comment]
Also, please squash commits into a single commit. Thanks!

File: apps/gallery/test/marionette/info_view_test.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var Gallery = require('./lib/gallery.js'),
+    Fullscreen_View = require('./lib/fullscreen_view.js'),
+    GalleryInfoView = require('./lib/gallery_info_view.js'),
+    assert = require('assert');
+
+marionette('the gallery', function() {
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/info_view_test.js]: nit: 'the gallery' can be changed to more informative text such as 'Gallery Info View tests'

File: apps/gallery/test/marionette/lib/gallery_info_view.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    _trimTextContent: function(text) {
+      // for some reason these unicode delimiters are inserted
+      // in the text node
+      return text.trim().replace(/[\u2068\u2069]/g, '');
+    },
+    
+    /**
+    * @return {Boolean} display status of info view.
+    */
+   waitForInfoViewDisplayed: function() {
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/lib/gallery_info_view.js]: nit: indentation

File: apps/gallery/test/marionette/info_view_test.js (hunk 2 above)
Comment (npark-mozilla) [inline apps/gallery/test/marionette/info_view_test.js]: oh actually, apparently displayed property does the wait call inside:
https://github.com/npark-mozilla/gaia/blob/1242615/apps/gallery/test/marionette/lib/fullscreen_view.js#L89

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=129e164a5022ea89263009cac1a5103aa9dcd994

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4a255fa17c28f5d0e3bfea8ec7d1aea2bc2b2877

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dbbc19938b8294cadc6e38f3447f427d0384e679

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=325ee44793d3a8458fdb13c681f68390c731ecd0

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a8e3e0a4204da90a3945d17e5abb0e576ba1bc84

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ecd88bfa718c9b26cd6799d15833cb3986764e02

**Pull request merged**